### PR TITLE
js: added EditPrintArea and AddPrintArea uno commands in classic and NB

### PR DESCRIPTION
**Pull request by Rash419**
updated unocommand.js using unocommand.py

Signed-off-by: Rash419 <rashesh.padia@collabora.com>
Change-Id: I0d972a16d5e9365af3ce31a39ceb006eb217627f

* Resolves: # 
* Target version: master 

### Summary
Apparently EditPrintArea is already async 

 Tested using this method:
1. user A opens dialog
2. user B opens the same dialog
3. user A does some action in dialog and clicks “OK”
4. user B does some action in dialog and clicks “OK”

Everything looks good!

### CODE REVIEW

File: browser/src/unocommands.js [hunk 1]
```diff
@@ -291,7 +293,7 @@ var unoCommandsArray = {
 	'InsertSlideTitleField':{presentation:{menu:_('Slide Tit~le'),},},
 	'InsertSlidesField':{presentation:{menu:_('Slide ~Count'),},},
 	'InsertSoftHyphen':{global:{menu:_('Insert s~oft Hyphen'),},},
-	'InsertSparkline':{spreadsheet:{menu:_('Insert Sparkline...'),},},
+	'InsertSparkline':{spreadsheet:{menu:_('Sparkline...'),},},
```
Comment (eszkadev) [inline browser/src/unocommands.js:296]: shouldn't be changed

Comment (timar) [reply browser/src/unocommands.js:296]: It's fine, the string was changed in core.
```
commit 273ebcc95defc82a9ec4073f8c0a951a9da89707
Author: Julien Nabet <serval2412@yahoo.fr>
Date:   Mon Jul 11 11:21:56 2022 +0200

    tdf#149738: Minor inconsistencies in sparkline related UI strings
    
    Change-Id: I5b4c31e69743ad519a8bcc99fa3a5a7512a4cc18
    Reviewed-on: https://gerrit.libreoffice.org/c/core/+/136955
```

**Comment by eszkadev** [comment]
Edit dialog is not async. It doesn't allow to open 2 instances of that dialog in different sessions. Also when opened in one session, second user has disabled many functions like font selector...

**Review by eszkadev — APPROVED** [review]

**Pull request merged**